docs/guides/notebooks/configuration/groups.rst

```diff
@@ -0,0 +1,129 @@
+############################
+User groups for file sharing
+############################
+
+Create and manage closed user groups in order to share private files in the ``/project`` directory.
+
+.. _user-group-create:
+
+Create a closed group
+=====================
+
+Comanage webpage
+----------------
+
+* In a browser, navigate to `id.lsst.cloud <https://id.lsst.cloud>`_ and log in.
+
+* In the left menu sidebar, click on "Groups" and then "My Groups".
+
+* At right, a list of all the groups associated with your account will appear.
+
+* In the top horizontal menu, click on "Add Group".
+
+* Set the group properties:
+
+  * The group name should be short and must start with ``g_``.
+
+  * Write a short description.
+
+  * Leave the status as the default "Active".
+
+  * Leave the checkbox next to "Open" unselected, to create a closed group.
+
+  * Leave the checkbox next to "Require All for Nested Memberships" unchecked; this can be changed later if nested sub-groups are created.
```

Comment:
I actually haven't tested nested subgroups. Does it work? If not let me know and we can try and ask for it to be hidden from the UI.

Reply:
It seems to work yes -- I was able to add g_cst as a nested source group for g_mlg_test, and then vice versa. So I think it can remain visible.